jupyter_notebook/static/notebook/js/cell.js

```diff
@@ -73,14 +73,23 @@ define([
             }
         });
 
+        // backward compat.
+        Object.defineProperty(this, 'cm_config', {
+            get: function() {
+                console.warn("warning you are accessing a deprecated field")
```

Comment:
Does console.warn() say which file & line the warning came from? If not, this message should be more specific.